docs/api.rst

```diff
@@ -13695,12 +13697,44 @@ Managing calendar reminders
       Content-Type: application/json
 
       {
-        "result": true,
-        "message": ""
+        "message": "",
+        "result": {
+          "eth_on_gas": "0.21760082633497774",
+          "gnosis_max_payments_by_currency": {
+              "AED": "36.0",
+              "EUR": "120.17",
+              "THB": "13615.25",
+              "USD": "85.0"
+          },
+          "top_days_by_number_of_transactions": [
+              {
+                "amount": 5,
+                "timestamp": 1716715671
+              }
+          ],
+          "trades_by_exchange": {
+              "bybit": 4,
+              "coinbase": 21,
+              "kraken": 35
+          },
+          "transactions_per_chain": {
+              "ARBITRUM_ONE": 1284,
+              "BASE": 463,
+              "ETHEREUM": 144,
+              "GNOSIS": 1067,
+              "OPTIMISM": 283,
+              "POLYGON_POS": 186,
+              "SCROLL": 17
+          }
+        },
+        "status_code": 200
       }
 
-  :resjson bool result: Boolean denoting success or failure.
+  :resjson string eth_on_gas: Amount of ETH spent on gas.
+  :resjson object gnosis_max_payments_by_currency: Object that maps currency symbol to the max amount paid in a single payments using that currency.
```

Comment:
you have this two times no? `gnosis_max_payments_by_currency` ?

Is this only for gnosis pay?

Reply:
yes appears twice. There is a entry only for gnosis pay